recipes/stringtie/1.2.4/meta.yaml

```diff
@@ -0,0 +1,25 @@
+about:
+  home: http://ccb.jhu.edu/software/stringtie/
+  license: Artistic License 2.0
+  summary: Transcriptome assembly and quantification for RNA-seq
+build:
+  number: 0
+  skip: True # [osx]
+package:
+  name: stringtie
+  version: 1.2.4
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
```

Comment:
As a reference, this # [osx] is a template language, it will be stripped out if you are not on OSX. So it has no influence in the end. It is just not a good "coding" style. On the other hand if we leave it in the guy who cares about OSX builds later on has less work :)